extensions/chat/standalone.mjs

```diff
@@ -24,9 +23,16 @@ class Standalone {
   async initialize() {
   }
 
-  c2s_broadcastMessage(player, arg){
+  async c2s_broadcastMessage(player, arg){
     // TODO: Check whether message is command like /help
-    this.helper.broadcastToAllUser(arg);
+    arg['msg_from'] = player.playerID;
+    await this.helper.broadcastToAllUser(arg);
+  }
+
+  async c2s_privateMessage(player, arg){
+    arg['msg_from'] = player.playerID;
+    await this.helper.callS2cAPI(arg.msg_to, 'chat', 'getPrivateMessage', 5000, arg);
```

Comment:
Optional: You can do both at the same time with Promise.all().
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all
```
await Promise.all([this.helper.callS2cAPI(...), this.helper.callS2cAPI(...)])
```